packages/queries/src/getColumns.ts

```diff
@@ -0,0 +1,33 @@
+import { DbSchema } from './types/schemas';
+
+export function getColumns(schema: DbSchema, table: string) {
+    const select: Record<string, true> = {};
+
+    const properties = schema.properties;
+
+    const tableDef = properties[table];
+
+    const tableDefProperties = tableDef.properties;
+
+    const columnsDef = tableDefProperties.columns;
+
+    const columnsDefProperties = columnsDef.properties;
+
+    for (const column in columnsDefProperties) {
```

Comment:
This can be simplified using Object.values

 `for (const columnDef of Object.values(columnsDefProperties))`

This avoids the `if (column)` check, and avoids the `columnDef = columnsDefProperties[column]`